staging/src/k8s.io/legacy-cloud-providers/azure/azure.go

```diff
@@ -727,8 +727,8 @@ func (az *Cloud) SetInformers(informerFactory informers.SharedInformerFactory) {
 		UpdateFunc: func(prev, obj interface{}) {
 			prevNode := prev.(*v1.Node)
 			newNode := obj.(*v1.Node)
-			if newNode.Labels[v1.LabelZoneFailureDomain] ==
-				prevNode.Labels[v1.LabelZoneFailureDomain] {
+			if newNode.Labels[v1.LabelTopologyZone] ==
```

Comment:
This switched from beta to GA labels, it's probably safe but I think we probably want to switch this in a separate PR.

Reply:
If we want to do that I can just pop the last commit.  That's fair, why don't I do that.